setup-app.py

```diff
@@ -24,3 +77,96 @@
 
 # Now the app can be started
 call(["cf", "start", appname])
+
+# Tail the application log
+print("Parsing log files.")
+end_token = "cf stop %s" % appname  # Seeing this in the log means certs done
+log_pipe = Popen("cf logs %s --recent" % appname, shell=True,
+                 stdout=PIPE, stderr=PIPE)
+log_lines = log_pipe.stdout.readlines()
+print("Waiting for certs (could take several minutes)")
+while end_token not in ''.join(log_lines):
```

Comment:
Good catch. I'll add a ten-minute timeout with a message that instructs users to look at the logs for errors.